### ISSUE TITLE
Special character in data.frame causing error in upload

### ISSUE THREAD

**Issue by ksens** [Paradigm4/SciDB-Py#142]
@rvernica 

Having a special character in a pandas data.frame is causing an issue on upload. This was reported at a customer deployment. 

I have not had a chance to do any scidb-py debugging on this yet. Can you comment please? 

```py
import pandas as pd
from scidbpy import connect

# data.frame with one special character in the third column
project_df = pd.DataFrame({'projectid': 'test_project2', 'key': 'sometagkey', 'value': 'µsometagvalue'}, index=[0])

# upload fails
db = connect()
db.input('''<name: string not null, metadata_attrkey: string not null, metadata_value: string not null>[up_idx]''',
  upload_data=project_df[['projectid', 'key', 'value']].to_records(index=False)
).store()
```

Error is as follows:

```
requests.exceptions.HTTPError: 406 Client Error: UserException in file: src/query/ops/input/InputArray.cpp function: moveNext line: 192 instance: s0-i0 (0)
Error id: scidb::SCIDB_SE_IMPORT_ERROR::SCIDB_LE_FILE_IMPORT_FAILED
Error description: Import error. Import from file '/tmp/shim_input_buf_KJmiXS' (instance 0) to array 'tmp_input_array' failed at line 2, column 0, offset 51, value='(unreadable)': Failed to read file: 0. for url: http://localhost:8080/execute_query?query=store%28input%28%3Cname%3A+string+not+null%2C+metadata_attrkey%3A+string+not+null%2C+metadata_value%3A+string+not+null%3E%0A++++++++++++++++++++%5Bup_idx%5D%2C+%27%2Ftmp%2Fshim_input_buf_KJmiXS%27%2C+0%2C+%27%28string%2C+string%2C+string%29%27%29%2C+py_8dc7386aca5611eaacf1c4b301cb1651_1%29&id=ux1cd7zv9kj7kztobqcm0gm58p6kg08q
```

CC: @apoliakov 

## For my reference only

The application code that triggers the above error is a REST API call to upload project with tags consisting of special characters

```
curl -X POST "http://127.0.0.1:9050/api/v0/project" -H "accept: application/json" -H "Authorization: Bearer <TOKEN>" -H "Content-Type: application/json" -d "{ \"projectid\": \"test_project2\", \"tags\": { \"sometagkey\": \"µsometagvalue\" }}"
```

**Comment by apoliakov** [comment]
Hey @rvernica ! Could you please take some time to assist Kriti with this?

We should figure out if it's a SciDB-Py issue or maybe an issue with SciDB input() operator. If latter, we'll need to involve dev team. Thanks!

**Comment by rvernica** [comment]
Bug in SciDB-Py. With the fix, the upload works fine:
```
AFL% scan(py_276be760caf011ea9e778c705a1a00c0_1);
{up_idx} name,metadata_attrkey,metadata_value
{0} 'test_project2','sometagkey','µsometagvalue'
```

**Comment by ksens** [comment]
Seems like the Travis tests failed with this patch. Should I wait till you fix this?

**Comment by rvernica** [comment]
The tests pass now. The fix is in the #143 PR. Once merged, we can push a new package version if needed.

**Comment by ksens** [comment]
Thanks. I usually install directly from github so I can install once the PR is merged.
